modules/openapi-generator/src/main/java/org/openapitools/codegen/languages/AbstractCSharpCodegen.java

```diff
@@ -835,6 +836,9 @@ public OperationsMap postProcessOperationsWithModels(OperationsMap objs, List<Mo
                         }
                     }
 
+                    Set<CodegenParameter> referenceTypes = operation.allParams.stream().filter(p -> p.vendorExtensions.get("x-is-value-type") == null && !p.isNullable).collect(Collectors.toSet());
+                    operation.vendorExtensions.put("x-notNullableReferenceTypes", referenceTypes);
+                    operation.vendorExtensions.put("x-hasNotNullableReferenceTypes", referenceTypes.size() > 0);
```

Comment:
shall we make it consistent to use kebab-case (e.g. x-has-not-nullable-reference-types) ?

Reply:
done